techniques/failures/F46.html

```diff
@@ -84,8 +85,9 @@
       </section>
       <section class="results"><h3>Expected Results</h3>
          <ul>
-            <li>If any check above is false, then this failure condition applies
-                                    and the content fails the Success Criterion.</li>
+            <LI>If #1 is true, then this failure condition does not apply.</LI>
```

Comment:
I like lower case \<li> elements, but that's pretty trivial.  
  
I think there must be an easiier way to phrase this. Whether #1 is true or false, this fails the SC. I think this statement is what it is trying to say.

Reply:
I find the resulting 'double negative' a bit harder to parse in your suggestion.

The phrasing used in the test is the normal way such things are expressed in the failure techniques (although there is variation in the 100+ [failure techniques](https://www.w3.org/WAI/WCAG22/Techniques/#failures)).

Reply:
There's a lot of content in WCAG to maintain...  And ya, double negatives are annoying.   
  
Could it be simplified to:   
  "If #1 is true, then this does not apply."